umm-spec-lib/test/cmr/umm_spec/test/migration/version_migration.clj

```diff
@@ -1129,3 +1129,40 @@
             {:URLs ["www.contact.shoo.com"],
              :Description "Contact related url description"}]
            collection-contact-persons))))
+
+(deftest migrate-1-9-tiling-identification-systems-to-1-10
+  (let [tiling-id-systems {:TilingIdentificationSystems
+                           [{:TilingIdentificationSystemName "MISR"
+                             :Coordinate1 {:MinimumValue 1.0
+                                           :MaximumValue 10.0}
+                             :Coordinate2 {:MinimumValue 1.0
+                                           :MaximumValue 10.0}}
+                            {:TilingIdentificationSystemName "Heat Miser"
+                              :Coordinate1 {:MinimumValue 11.0
+                                            :MaximumValue 20.0}
+                              :Coordinate2 {:MinimumValue 11.0
+                                            :MaximumValue 20.0}}]}
+        result (vm/migrate-umm {} :collection "1.9" "1.10" tiling-id-systems)
+        other-result (vm/migrate-tiling-identification-systems tiling-id-systems)]
```

Comment:
I think other-result needs to be removed, right? 

And Siwei, Mark did not have to do a conversion from 1.1o to 1.9.